src/qmltypes/qmlfilter.h

```diff
@@ -145,7 +145,6 @@ public slots:
 signals:
     void presetsChanged();
     void analyzeFinished(bool isSuccess);
-    void changed(); /// Use to let UI and VUI QML signal updates to each other.
     void changed(QString name);
```

Comment:
Consolidate to `void changed(QString name = QString());`

Reply:
done